deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/deployment.yaml

```diff
@@ -68,6 +68,9 @@ spec:
           {{- if .Values.args.pre16k8sVersion }}
           - --k8sVersion=1.5
           {{- end }}
+          {{- if not .Values.args.cleanupSccImpersonationResources }}
```

Comment:
For your tests, please add evidence of resources getting left over when the flag is set to false.

Reply:
I've added steps 4&5 in the description to prove the resources are not created after restarting the kubeturbo pod